packages/ember/lib/index.js

```diff
@@ -306,6 +307,7 @@ import {
   collect
 } from 'ember-runtime';
 
+Ember.A = A;
```

Comment:
Why was this added?

Reply:
Nevermind, https://github.com/emberjs/ember.js/pull/16206/files#diff-7be349ee898133ade03bc4adca5582bcL154.